### PR TITLE
Typos

### LINKED ISSUE #66535: A Tale of Brigandines and typos
### Is your feature request related to a problem? Please describe.

Yet another typo batch, as usual, I'll add stuff here as I find it as long as nobody fixes them, but I once again reached a critical detected typo mass while translating, so I am exposing them to the public here.

### Solution you would like.

This special monster attack is obviously mistyped, but it's mistyped to the point where I'm not really that sure what would be the correct fix, so go wild I guess:
- [monster] stares at you, but you the wind bursts past you!

Nested group brigandine description got an unnecessary period:
- Recipes related to constructing various sizes of brigandines with shoulder guards.. -> Recipes related to constructing various sizes of brigandines with shoulder guards.

Unnecessary capitalization of these nested groups:
- Mild steel brigandine gloves -> mild steel brigandine gloves
- Medium steel brigandine gloves -> medium steel brigandine gloves
- High steel brigandine gloves -> high steel brigandine gloves
- Hardened steel brigandine gloves -> hardened steel brigandine gloves
- Tempered steel brigandine gloves -> tempered steel brigandine gloves

### Describe alternatives you have considered.

Not pointing out typos.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Procyonae**
#### Summary
None

#### Purpose of change
Fixes #66535

#### Describe alternatives you've considered
Moaning that there weren't enough typos

### CODE REVIEW

File: data/json/monster_special_attacks/monster_attacks.json [hunk 1]
```diff
@@ -323,9 +323,9 @@
     "damage_max_instance": [ { "damage_type": "bash", "amount": 20 } ],
     "hit_dmg_u": "%1$s stares at you intently and a sudden wind buffets you!",
     "hit_dmg_npc": "%1$s stares intently at <npcname> and a hurricane force wind hits them!",
-    "no_dmg_msg_u": "%1$s's wind buffets your %2$s, but it doesn't catch hold.",
-    "no_dmg_msg_npc": "%1$s's winds hit <npcname>, but it doesn't catch hold.",
-    "miss_msg_u": "%s stares at you, but you the wind bursts past you!",
+    "no_dmg_msg_u": "%1$s's wind buffets your %2$s, but it doesn't catch hold!",
```
Comment (Maleclypse) [inline data/json/monster_special_attacks/monster_attacks.json:326]: So I don't char about exclamation points or not but lets follow whatever the accepted standard is.

Comment (Procyonae) [reply data/json/monster_special_attacks/monster_attacks.json:326]: Oops I messed up rebasing good catch

Comment (Procyonae) [inline data/json/monster_special_attacks/monster_attacks.json:326]:
```suggestion
    "no_dmg_msg_u": "%1$s's wind buffets your %2$s, but it doesn't catch hold.",
```

File: data/json/monster_special_attacks/monster_attacks.json [hunk 2]
```diff
@@ -323,9 +323,9 @@
...
+    "no_dmg_msg_npc": "%1$s's winds hit <npcname>, but it doesn't catch hold!",
```
Comment (Procyonae) [inline data/json/monster_special_attacks/monster_attacks.json:327]:
```suggestion
    "no_dmg_msg_npc": "%1$s's winds hit <npcname>, but it doesn't catch hold.",
```

**Comment by sadenar** [comment]
Since this isn't merged yet, going to comment what I found today:
- talking animal pie (humanitarian pie) and prick pie (cannibal pie) do not seem to have plurals
- Protesting and poser pizza seem to have no plurals either, but honestly this would be a nice opportunity to rename them into something more Italian and in my opinion, pizza dei folletti (singular) / pizze dei folletti (plural) (goblin pizza) and pizza humana (singular) / pizze humane (plural) (human pizza)

**Comment by Procyonae** [comment]
The cannbial and "humanitarian" stuff is weird someone else can deal with those

**Pull request merged**